pkg/apis/kubevirt/v1beta1/nodemaintenance_types.go

```diff
@@ -1,4 +1,4 @@
-package v1alpha1
+package v1beta1
```

Comment:
API version change would need to be addressed in upgrades. You will probably have to add a webhook for that in a different PR.

Reply:
I am not sure that upgrade checks are done with the build process. So far I am not aware of such checks in other operators. Did you see upgrade checks in other projects?

Reply:
In any event addition of an upgrade check is a major change that in my opinion should be added with a different PR, and with this PR which is a fix to a previous change that broke the build.

Reply:
Adding automation tests for upgrades is one thing and could be indeed be added in the future in a different PR. 
Maybe I picked the API change a bit late but since up until now NMO was using only valpha1 there were no potential issues with upgrading. 
Now that the API changed there is a need to make sure that an upgrade procedure  (e.g. upgrading NMO  from v0.4.0 to v0.5.0) won't cause any errors with the updated version (taking into account the previous and new CRD, existing workloads in the system, etc..) 
There might not be an issue at all but this is worth looking into, see https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning/